.pre-commit-config.yaml

```diff
@@ -1,8 +1,4 @@
 ---
-
-default_language_version:
-  python: python3.8  # This should be set to the lowest Python version that we support.
```

Comment:
Removed the default language so that we are using the python version of the virtual environment. This will make creation of the development environment and the management of the CI easier. 

We will still run `pre-commit` with the lowest supported python version, currently python 3.8, in the CI

Reply:
I am not convinced by this. Feels like there would be discrepancy between behaviour locally and on the CI, it might alienate people (I know I would be annoyed if tests pass locally but not on the CI).